services/servo.md

```diff
@@ -2,15 +2,47 @@
 
     identifier: 0x12fc9103
 
-Servo is a small motor directed with a PWM signal.
-This services fixes the servo period at 20ms, and the pulse can be regulated.
+Servo is a small motor with arm that can be pointing at a specific direction.
 
 ## Registers
 
-    rw pulse: u32 us {typical_min = 500, typical_max = 2500} @ value
+    rw angle: i16.16 ° {typical_min = -60, typical_max = 60} @ value
 
-Specifies length of the pulse in microseconds. The period is always 20ms.
+Specifies the angle of the arm.
 
     rw enabled: bool @ intensity
 
-Turn the power to the servo on/off.
\ No newline at end of file
+Turn the power to the servo on/off.
+
+    rw offset?: i16.16 ° @ 0x81
+
+Correction applied to the angle to account for the servo arm drift.
+
+    const min_angle?: i16.16 ° @ min_reading
+
+Lowest angle that can be set.
+
+    const max_angle?: i16.16 ° @ max_reading
+
+Highest angle that can be set.
+
+    enum Variant: u32 {
+        PositionalRotation = 1,
+        ContinuousRotation = 2,
```

Comment:
should we drop the continuous servo here? it should use the motor service (in a given project, you can't really replace positional servo with a continuous one, but you can replace a DC motor with a continuous servo)